dist/index.js

```diff
@@ -16429,6 +16443,7 @@ async function main() {
       setPagesPath({ staticSiteGenerator, generatorConfigFile, path: siteUrl.pathname })
     }
     outputPagesBaseUrl(siteUrl)
+    core.exportVariable('GITHUB_PAGES', true)
```

Comment:
Same comment but for the built distributable version. 😊 

> Since environment variables are all strings, let's be a little more explicit with the data type here. 👍🏻 

```suggestion
    core.exportVariable('GITHUB_PAGES', 'true')
```